src/background/handlers/windows.ts

```diff
@@ -2,13 +2,20 @@ import { Command } from '../../types'
 
 const classifier = 'window'
 
+const group = {
+  icon: '⚙️',
+  label: 'Browser',
+  char: '>',
```

Comment:
This is the same char as chrome?

Reply:
Yes, that's intentional. I figured the windows and chrome commands could all fall under the same group. What do you think?

Reply:
hmm then why do they have different Icons? Wouldn't that lead to confusion?

Reply:
Oh, that's a mistake. Eh, let's fix it later.